tests/capi/unittest_capi_inference.cc

```diff
@@ -2812,6 +2812,30 @@ TEST (nnstreamer_capi_util, info_get_tsize_03_n)
   ASSERT_EQ (status, ML_ERROR_NONE);
 }
 
+/**
+ * @brief Test utility functions (public)
+ */
+TEST (nnstreamer_capi_util, info_clone)
+{
+  gint status;
+  ml_tensors_info_h in_info, out_info;
+
+  status = ml_tensors_info_create (&in_info);
+  EXPECT_EQ (status, ML_ERROR_NONE);
+  status = ml_tensors_info_create (&out_info);
+  EXPECT_EQ (status, ML_ERROR_NONE);
+  status = ml_tensors_info_set_count (in_info, 1);
+  EXPECT_EQ (status, ML_ERROR_NONE);
+
+  status = ml_tensors_info_clone (out_info, in_info);
+  EXPECT_EQ (status, ML_ERROR_NONE);
```

Comment:
How about adding extra test? like
```
  ml_tensors_info_get_count (out_info, &count);
  EXPECT_EQ (count, 1);
``

Reply:
Thanks! I needed to set type and dimension also. Updated.